### PR TITLE
LCFS-1280: Validation for Quantity Supplied

### PR DESCRIPTION
**Pull request by areyeslo**
- The default **Quantity supplied** value being 0 should not be allow. 
- Zero should not be a valid amount in a fuel supply record and the row should not save until the user has entered a value.

PR related: https://github.com/bcgov/lcfs/pull/1312

[Story](https://github.com/orgs/bcgov/projects/137/views/1?pane=issue&itemId=88333582&issue=bcgov%7Clcfs%7C1280)

### CODE REVIEW

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
Looks good, I would just like to see a frontend test covering this 0 quantity state and that it shows the error correctly.

**Comment by AlexZorkin** [comment]
Let's get this merged, just need to fix the conflicts, otherwise looks good to go.

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
We don't want to add this validation across quantity fields, i think just focus on fuel_supply. If you hear differently from business area then let me know.

File: backend/lcfs/web/api/notional_transfer/schema.py [hunk 1]
```diff
@@ -20,7 +20,9 @@ class NotionalTransferCreateSchema(BaseSchema):
     address_for_service: str
     fuel_category: str
     received_or_transferred: ReceivedOrTransferredEnumSchema
-    quantity: int
+    quantity: int = Field(
```
[the remark this answers is not in the document]
Comment (areyeslo) [reply backend/lcfs/web/api/notional_transfer/schema.py:23]: Let me check with Lindy and update the ticket if applicable.

File: backend/lcfs/web/api/transaction/schema.py [hunk 2]
```diff
@@ -71,7 +71,9 @@ class TransactionViewSchema(BaseSchema):
     transaction_type: str
     from_organization: Optional[str] = None
     to_organization: str
-    quantity: int
+    quantity: int = Field(
```
[the remark this answers is not in the document]
Comment (areyeslo) [reply backend/lcfs/web/api/transaction/schema.py]: Removed this validation

File: backend/lcfs/web/api/notional_transfer/schema.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (areyeslo) [reply backend/lcfs/web/api/notional_transfer/schema.py]: Removed this validation

**Review by AlexZorkin — APPROVED** [review]

**Pull request merged**